DEVELOPER_GUIDE.md

```diff
@@ -275,6 +279,13 @@ This method can also be used to develop against the [full distribution of OpenSe
 
 _This step is only mandatory if you have the [`security` plugin](https://github.com/opensearch-project/security) installed on your OpenSearch cluster with https/authentication enabled._
 
+> Running `yarn opensearch snapshot -P <security_plugin_url>` will not work since the plugin needs some additional configuration to setup correctly. To configure it correctly, here are the steps:
+>
+> 1. Run `yarn opensearch snapshot -P <security_plugin_url>`. This will fail to run and complain about missing certificates. Ignore that since what we want is for the build artifacts
+> 2. Run `export initialAdminPassword=<inital admin password>` since its needed by the configuration script
```

Comment:
should we provide just an example, like it can be `admin` or imply that this is setting up initial admin password not so much that it has to be preconfigured

Reply:
I'm okay with that. This is one of those docs where I want this to be a stub and see where people get confused and need more information. I think the variable name is quite self explanatory ;)